plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -1808,82 +1911,13 @@ public function update_meta( &$object, $meta ) {
 		return $this->data_store_meta->update_meta( $object, $meta );
 	}
 
-	/**
-	 * Returns list of metadata that is considered "internal".
-	 *
-	 * @return array
-	 */
-	public function get_internal_meta_keys() {
-		// XXX: This is mostly just to trick `WC_Data_Store_WP` for the time being.
-		return array(
-			'_customer_user',
-			'_order_key',
-			'_order_currency',
-			'_billing_first_name',
-			'_billing_last_name',
-			'_billing_company',
-			'_billing_address_1',
-			'_billing_address_2',
-			'_billing_city',
-			'_billing_state',
-			'_billing_postcode',
-			'_billing_country',
-			'_billing_email',
-			'_billing_phone',
-			'_shipping_first_name',
-			'_shipping_last_name',
-			'_shipping_company',
-			'_shipping_address_1',
-			'_shipping_address_2',
-			'_shipping_city',
-			'_shipping_state',
-			'_shipping_postcode',
-			'_shipping_country',
-			'_shipping_phone',
-			'_completed_date',
-			'_paid_date',
-			'_edit_lock',
-			'_edit_last',
-			'_cart_discount',
-			'_cart_discount_tax',
-			'_order_shipping',
-			'_order_shipping_tax',
-			'_order_tax',
-			'_order_total',
-			'_payment_method',
-			'_payment_method_title',
-			'_transaction_id',
-			'_customer_ip_address',
-			'_customer_user_agent',
-			'_created_via',
-			'_order_version',
-			'_prices_include_tax',
-			'_date_completed',
-			'_date_paid',
-			'_payment_tokens',
-			'_billing_address_index',
-			'_shipping_address_index',
-			'_recorded_sales',
-			'_recorded_coupon_usage_counts',
-			'_download_permissions_granted',
-			'_order_stock_reduced',
-		);
-	}
-
 	/**
 	 * Returns keys currently handled by this datastore manually (not available through order properties).
 	 *
 	 * @return array List of keys.
 	 */
 	protected function get_internal_data_store_keys() {
```

Comment:
This method I added while we were finalizing the design, but it's not not a general datastore method but something specific of this class. I believe it should be safe to get rid of it (and the only place where it's used in this class) as ideally there won't be datastore-specific order props (that's exactly what we're fixing here 😅).

Reply:
👍 removed in https://github.com/woocommerce/woocommerce/pull/34627/commits/ec0afd608441279ef88e7e2748959791a021521d